pubsub/topics/main.go

```diff
@@ -145,6 +154,43 @@ func publish(client *pubsub.Client, topic, msg string) error {
 	return nil
 }
 
+func publishThatScales(client *pubsub.Client, topic string, n uint64) error {
+	ctx := context.Background()
+	var wg sync.WaitGroup
+	var n_err uint64
+	// [START pubsub_publish_with_error_handling_that_scales]
```

Comment:
Let's have this just below the `ctx` definition, the `wg` object might not be obvious and it plays an important role in the synchronization.